crowdfunding_compassion/templates/myaccount_crowdfunding_project_update_page.xml

```diff
@@ -0,0 +1,32 @@
+<odoo>
+    <data>
+        <template id="project_update_view_template">
```

Comment:
This is exactly the same template as `participant_update_view`. You can rename it to more generic name like `crowdfunding_form_template` and use the same template for the two pages. The only difference will be in the controller where you will inject another form in the template.